src/components/decision/css/style.css

```diff
@@ -27,24 +27,9 @@
     */
     pointer-events: none;
   }
-
-  &:active:not(.cancel-next),
-  &.show-decision-color {
-    &:not(.disabled, :disabled) {
-      outline: none;
-      border: none;
-
-      border: var(--oe-border-width) solid var(--decision-color);
-      --background-color: var(--decision-color);
-    }
-  }
 }
 
-/*
-  I declare the --decision-color variable on the decision button class not
-  on the id selector because I need the specificity to be 0,1,0 so that I can
-  override the variable with a decision class
- */
-.decision-button {
-  --decision-color: none;
+.decision-color-pill {
+  padding: 0.2em;
```

Comment:
don't pad vertically

Reply:
Discussed offline, padding has been removed and replaced with height